File: 125-valid-palindrome/valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """

        nStr = ""
        for i in s:
            if i.isalnum():
                nStr += i.lower()
        return nStr == nStr[::-1]
        

File: 125-valid-palindrome/valid-palindrome.cpp
#include <iostream>

class Solution {
public:
    bool isPalindrome(string s) {
        string str = "";
        for (int i = 0; i < s.size(); i++){
            if (isalnum(s[i])){
                str += tolower(s[i]);
            }
        }
        string st = str;
        reverse(str.begin(), str.end());
        if (str == st){
            return true;
        }
       return 0;
    }
};